src/workers/queueScraper.worker.ts

```diff
@@ -1,15 +1,23 @@
 import { Worker, Queue } from 'bullmq';
 
+import Redis from '@/config/redis';
 import { AsyncProcess } from '@/constants/enum';
 import { BullQueue } from '@/lib/worker';
 
 import buildScraper from './queueScraper';
 
-const scrapeBullQueue = new Queue(AsyncProcess.QUEUE_SCRAPE);
+const redis = Redis.getInstance();
```

Comment:
Let's be more descriptive, especially as there is another variable close to this one (`Redis`):

```suggestion
const redisConnection = Redis.getInstance();
```

Reply:
Fixed.